src/donate.js

```diff
@@ -9,18 +9,124 @@ window.WebMonetizationScripts.paymentPointerToUrl = function (paymentPointer) {
   const parsed = new URL('https://' + paymentPointer.substring(1))
   return parsed.origin + (parsed.pathname || '/.well-known/pay')
 }
+const css = `
+  #coil-advert-widget {
+    position: fixed;
+    z-index: 10000;
+  }
+
+  #wm-widget {
+    pointer-events: auto;
+    line-height: normal;
+    vertical-align: middle;
+    display: inline-block;
+    position: relative;
+    bottom: 0;
+    left: 0;
+    cursor: pointer;
+
+  }
+  #wm-widget-text {
+    vertical-align: middle;
+    line-height: normal;
+    margin-left: 4px;
+    padding-top: 7px;
+    padding-bottom: 5px;
+    padding-left: 14px;
+    padding-right: 14px;
+    box-sizing: border-box;
+    height: 32px;
+    border-radius: 16px;
+    background-color: #2D333A;
+    color: #fff;
+    font-size: 14px;
+    opacity: 0;
+    transition: opacity 0.2s;
+    font-family: Roboto, sans-serif;
+  }
+
+  #wm-link, #wm-link:hover, #wm-link:active {
+    color: white;
+  }
+`
+
+
+window.WebMonetizationScripts.createAdvertWidget = function () {
```

Comment:
Do you think we should do this as a popup, or should we provide a component for the site to include in its layout? Kinda like the little 'donate with paypal' button that some sites have.

Reply:
I think we can have both. I can make another one which works more like the 'donate with paypal' button